Description: Removes the @javax.annotation.Generated annotation in the template to fix the build failure with Java 11
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: no
--- a/hppc-template-processor/src/main/java/com/carrotsearch/hppc/generator/TemplateOptions.java
+++ b/hppc-template-processor/src/main/java/com/carrotsearch/hppc/generator/TemplateOptions.java
@@ -97,10 +97,10 @@
 
   public String getGeneratedAnnotation() {
     return String.format(Locale.ROOT, 
-        "@javax.annotation.Generated(\n" + 
-        "    date = \"%s\",\n" +
-        "    value = \"%s\")",
+        "//@javax.annotation.Generated(\n" + 
+        "//    date = \"%s\",\n" +
+        "//    value = \"%s\")",
         getTimeNow(),
         TEMPLATE_FILE_TOKEN);
   }
-}
\ No newline at end of file
+}
